creator/lib/screens/salon_application_screen.dart

```diff
@@ -69,15 +69,18 @@ class _SalonApplicationScreenState extends State<SalonApplicationScreen> {
           ListTile(
             title: Text('ログアウト'),
             onTap: () async {
-              print('ログアウトボタンを押した');
-              await FirebaseAuth.instance.signOut();
-              await Navigator.of(context).pushReplacement(
-                MaterialPageRoute(
-                  builder: (context) {
-                    return LoginScreen();
-                  },
-                ),
-              );
+              final user = FirebaseAuth.instance.currentUser;
+              if (user != null) {
```

Comment:
```suggestion
              if (FirebaseAuth.instance.currentUser != null) {
```

`final user` is not needed

Reply:
Thanks, I'll fix it.